build.sh

```diff
@@ -174,3 +174,7 @@ if [ "$format_error_files" != "" ]; then
   exit 42
 fi
 
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
```

Comment:
Sorry for the previous comment, my ignorance! After reading [the docs](http://docs.travis-ci.com/user/environment-variables/#Convenience-Variables) I realize that we must test against `"false"` here.